editor/plugins/node_3d_editor_gizmos.cpp

```diff
@@ -3767,19 +3767,19 @@ int VoxelGIGizmoPlugin::get_priority() const {
 String VoxelGIGizmoPlugin::get_handle_name(const EditorNode3DGizmo *p_gizmo, int p_id, bool p_secondary) const {
 	switch (p_id) {
 		case 0:
-			return "Extents X";
+			return "Size X";
 		case 1:
-			return "Extents Y";
+			return "Size Y";
 		case 2:
-			return "Extents Z";
+			return "Size Z";
 	}
 
 	return "";
 }
 
 Variant VoxelGIGizmoPlugin::get_handle_value(const EditorNode3DGizmo *p_gizmo, int p_id, bool p_secondary) const {
 	VoxelGI *probe = Object::cast_to<VoxelGI>(p_gizmo->get_node_3d());
-	return probe->get_extents();
+	return probe->get_size() / 2;
```

Comment:
```suggestion
	return probe->get_size();
```
Same reason as above, if the label says size this should return size not extents